src/program/services/downloaders/models.py

```
@@ -1,5 +1,5 @@
 from datetime import datetime
-from typing import Dict, List, Literal, Optional, Union
+from typing import Any, Dict, List, Literal, Optional, Union
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)